charts/vc-authn-oidc/README.md

```diff
@@ -1,6 +1,6 @@
 # VC-AuthN OIDC
 
-![Version: 0.1.5](https://img.shields.io/badge/Version-0.1.5-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 2.0.0-rc2](https://img.shields.io/badge/AppVersion-2.0.0--rc2-informational?style=flat-square)
+![Version: 0.1.7](https://img.shields.io/badge/Version-0.1.5-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 2.0.0](https://img.shields.io/badge/AppVersion-2.0.0-informational?style=flat-square)
```

Comment:
Please update the chart version in the URL as well. 
`https://img.shields.io/badge/Version-0.1.5-informational?style=flat-square` should be `https://img.shields.io/badge/Version-0.1.7-informational?style=flat-square`